streaming/src/main/scala/org/apache/spark/streaming/scheduler/JobSet.scala

```diff
@@ -62,12 +64,21 @@ case class JobSet(
   }
 
   def toBatchInfo: BatchInfo = {
+    val failureReasons: Map[Int, String] =
```

Comment:
I generally prefer multiline statements like this to be put in a `{... }` . Becomes easier to read.